src/ledger/test/LedgerTestUtils.cpp

```diff
@@ -235,6 +235,11 @@ makeValid(ClaimableBalanceEntry& c)
 
     c.asset.type(ASSET_TYPE_CREDIT_ALPHANUM4);
     strToAssetCode(c.asset.alphaNum4().assetCode, "CAD");
+
+    if (c.ext.v() == 1)
+    {
+        c.ext.v1().flags &= MASK_CLAIMABLE_BALANCE_FLAGS;
```

Comment:
this is not valid. `flags == 0` iif `v() == 0`. So this line should be `c.ext.v1().flags = MASK_CLAIMABLE_BALANCE_FLAGS`

Reply:
Done.